### ISSUE TITLE
Maybe should be a private repository?

### ISSUE THREAD

**Issue by russplaysguitar** [jdstone/csci340#1]
You don't want anyone copying your homework assignments!

**Comment by jdstone** [comment]
Yeah, I know. This is only temporary. It was private, I just made it public for a little bit.  Thanks for watching out for me! :smiley:
